docs/sources/write/style-guide/inclusive-writing/index.md

```diff
@@ -95,3 +95,14 @@ Use the following approach to describe relationships between nodes or processes:
 - Use _secondary_, _replica_, or _child_, instead of _slave_.
 
 Avoid _master_ or _slave_.
+
+### Right and left
```

Comment:
Do we want to say anything about use of the word "hand" as a qualifier, as in "top-right hand side"? Probably not, just want to be sure.

Reply:
It's already in the Vale linter since this is an easier rule to implement, but it probably doesn't hurt to add it.